### ISSUE TITLE
Implement new homepage hero design

### ISSUE THREAD

**Issue by globalmatt** [Charity-Shop-Exchange/Charity-Shop-Exchange#36]
<img width="1081" alt="Screenshot 2020-03-26 at 18 58 02" src="https://user-images.githubusercontent.com/885411/77837674-9f130880-71b7-11ea-9656-225f3c7d8c9e.png">

**Comment by kelvinlockwood** [comment]
This design isn't ready for production just yet, waiting on Greta at Drew for an updated version @dickyjiang

**Comment by kelvinlockwood** [comment]
This is now a duplicate of #51
